src/components/events/createEvent/basicDetails.tsx

```diff
@@ -6,12 +6,12 @@ import {
   Typography,
 } from '@mui/material';
 import SelectField from '../../general/selectField';
-import DateRangePicker from '../commonEventFields/dateRangePicker';
+import DateRangePicker from '../../general/dateRangePicker';
 import { BLOCKS, EVENT_TYPES } from '../../../types/enums';
 
 interface DetailsProps {
-    currDates: Date[];
-    setCurrDates: (param: Date[]) => void;
+    currDates: string[];
```

Comment:
i guess there was a good reason to switch from date to string?

Reply:
Yes.
This is a component that is used in several places, including in the Stars filters.
There is a data structure called "filtersData" that accepts only string states and handles the filtering, so it was easier to convert this compoent to string than it would be to convert to filtersData to accept Dates.